graphql/src/store/query.rs

```diff
@@ -141,6 +149,14 @@ fn build_fulltext_filter_from_object(
     )
 }
 
+fn resolve_type_name(ty: &Type) -> &String {
+    match ty {
+        Type::ListType(inner) => resolve_type_name(inner),
+        Type::NonNullType(inner) => resolve_type_name(inner),
+        Type::NamedType(name) => name,
+    }
+}
```

Comment:
This is `get_base_type` from `graph/src/data/graphql` (just do something like `use graph::data::graphql::TypeExt as _` to be able to use it)

Reply:
866883c02f2c310f724cf7555939ea8a31547382